manifests/00-ingress-credentials-request.yaml

```diff
@@ -26,6 +26,7 @@ spec:
       - route53:ListTagsForResources
       - route53:ChangeResourceRecordSets
       - tag:GetResources
+      - sts:AssumeRole
```

Comment:
Let's remove this a la:
https://github.com/openshift/cluster-ingress-operator/pull/972

Reply:
yep done.